siriuspy/siriuspy/devices/orbit_interlock.py

```diff
@@ -124,6 +125,10 @@ def _mean(var1, var2):
     def _diff(var1, var2):
         return var1 - var2
 
+    @staticmethod
```

Comment:
do you really need to define this method? couldn't you pass the builtin function `min` to the dictionary?

Reply:
done!